docs/app/templates/getting-started/setup.hbs

```diff
@@ -203,9 +170,18 @@ for more information about including assets into your app.
 
 <p>These options can be set in your <code>ember-cli-build.js</code> file:</p>
 
-{{! template-lint-disable block-indentation }}
+{{! prettier-ignore }}
 <pre class="highlight"><code>//your-bootstrap-app/ember-cli-build.js
-    module.exports = function(defaults) { let app = new EmberApp(defaults, {
-    'ember-bootstrap': { importBootstrapCSS: false, exclude: ['bs-popover',
-    'bs-accordion'] } }); return app.toTree(); };
-  </code></pre>
\ No newline at end of file
+
+module.exports = function(defaults) {
+  const app = new EmberApp(defaults, {
+    'ember-bootstrap': {
+      bootstrapVersion: 4,
+      importBootstrapCSS: false,
+      exclude: ['bs-popover', 'bs-accordion'],
```

Comment:
Not related to blueprints, but will this `excludes/includes` structure work in a V2 addon?

Reply:
No. It won't work. But it's also not needed anymore in modern Embroider apps. Embroider only pulls those components into the build which are actually used. No need to provide a list manually. We should deprecate the feature and recommend relying on Embroider's tree shaking capabilities instead.